### PR TITLE
[DC-2395] Replace all bq.create_tables(...) calls in curation

### PR DESCRIPTION
**Pull request by lledezma**

### CODE REVIEW

**Review by nishanthpp93 — COMMENTED** [review]
Overall looks good, let me know what you think about the suggestion.

File: data_steward/cdr_cleaner/cleaning_rules/unit_normalization.py [hunk 1]
```diff
@@ -174,16 +174,14 @@ def setup_rule(self, client=None):
             the load job fails. Error raised from bq.upload_csv_data_to_bq_table
             helper function.
         """
+        if not client:
+            raise RuntimeError("Specify BigQueryClient object")
```
[the remark this answers is not in the document]
Comment (nishanthpp93) [reply data_steward/cdr_cleaner/cleaning_rules/unit_normalization.py:178]: Ideally you shouldn't need a check for that either. It should just conform to the setup_rule in existing cleaning rules. Do you want to make this change in this PR or a different ticket?

[the remark this answers is not in the document]
Comment (lledezma) [reply data_steward/cdr_cleaner/cleaning_rules/unit_normalization.py:178]: I'll make the change in this PR.

**Review by nishanthpp93 — APPROVED** [review]
Thanks for updating, looks good!

**Pull request merged**